blocklib/picoscope/test/qa_PicoscopePerformanceMonitor.cc

```diff
@@ -94,7 +94,7 @@ int main(int argc, char* argv[]) {
         expect(eq(ConnectionResult::SUCCESS, graph.connect<"out", 7>(ps).template to<"in">(sinkH)));
     }
 
-    auto sched                                        = scheduler::Simple{std::move(graph), threadPool};
```

Comment:
The `threadPool` definition should be removed and -- if necessary -- the default CPU-load processing thread pool limited to the number of min/max threads required. Otherwise OK.